### PR TITLE
tests: add details in tests for some snap subcommands

### PR DESCRIPTION
**Pull request by valentindavid**

### CODE REVIEW

File: tests/main/snap-download/task.yaml [hunk 1]
```diff
@@ -1,5 +1,13 @@
 summary: Check that snap download works
 
+details: |
+    `snap download` is a command to download snaps from the store.
+    This test verifies that snaps are downloaded alng with correct
```
Comment (sergiocazzolato) [inline tests/main/snap-download/task.yaml:5]: alng
```suggestion
    This test verifies that snaps are downloaded along with correct
```

File: tests/main/snap-download/task.yaml [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+    assertions. It verifies some command line options such as
+    `--edge`, `--beta`, `--target-directory`, `--basename` and
+    `--cohort`.  It makes sure that normal users (non-root) can also
+    download.
```
Comment (sergiocazzolato) [inline tests/main/snap-download/task.yaml:9]:
```suggestion
    download snaps.
```

**Review by sergiocazzolato — APPROVED** [review]
thanks

**Review by MiguelPires — APPROVED** [review]
thank you

**Pull request merged**